torch/linalg/__init__.py

```diff
@@ -830,35 +880,36 @@
 """)
 
 vector_norm = _add_docstr(_linalg.linalg_vector_norm, r"""
-linalg.vector_norm(input, ord=None, dim=None, keepdim=False, *, dtype=None, out=None) -> Tensor
+linalg.vector_norm(A, ord=None, dim=None, keepdim=False, *, dtype=None, out=None) -> Tensor
+
+Computes the vector norm of a vector.
```

Comment:
"a vector norm"? Since the function offers a choice of norms?

Reply:
Corrected here and in `norm`.